jadepix1/conf/jadepix1_without_electric_field_main.conf

```diff
@@ -43,13 +43,14 @@ adc_smearing = 0e
 adc_slope = 3.8e
 
 [DetectorHistogrammer]
+name = "mydetector"
 
 # replace by [VisualizationGeant4] to run the visualization
 #[Ignore]
 #mode = "gui"
 
 [ROOTObjectWriter]
-file_name = "data_withoutelectricfield.root"
+file_name = "data_without_electric_field.root"
```

Comment:
This should be with default electric filed

Reply:
By default, detectors do not have an electric ﬁeld applied. 
https://project-allpix-squared.web.cern.ch/project-allpix-squared/usermanual/allpix-manualch7.html#x8-660007 (chapter 7.6)

Reply:
@frankscu Can you confirm? Thanks!